### ISSUE TITLE
bower.json version not updated?

### ISSUE THREAD

**Issue by mla** [jedfoster/Readmore.js#82]
I'm new to bower, but when I try "bower install readmore" I get this mismatch error:

```shell
$ bower install readmore
bower resolve       git://github.com/jedfoster/Readmore.js.git#*
bower download      https://github.com/jedfoster/Readmore.js/archive/v2.0.1.tar.gz
bower extract       readmore#* archive.tar.gz
bower mismatch      Version declared in the json (2.0.0) is different than the resolved one (2.0.1)
bower resolved      git://github.com/jedfoster/Readmore.js.git#2.0.1
```

**Comment by jedfoster** [comment]
ee4e0f7 should fix this.

**Comment by artivilla99** [comment]
Getting a similar error:
```unix
bower readmore#*            not-cached git://github.com/jedfoster/Readmore.js.git#*
bower readmore#*               resolve git://github.com/jedfoster/Readmore.js.git#*
bower readmore#*              download https://github.com/jedfoster/Readmore.js/archive/v2.0.5.tar.gz
bower readmore#*               extract archive.tar.gz
bower readmore#*              mismatch Version declared in the json (2.0.4) is different than the resolved one (2.0.5)
bower readmore#*              resolved git://github.com/jedfoster/Readmore.js.git#2.0.5
bower Readmore.js#~2.0.5       install Readmore.js#2.0.5
```
